server/server.go

```diff
@@ -212,7 +214,7 @@ func (s *Server) Serve(ctx context.Context, httpAddr, grpcAddr string) error {
 	httpServer := &http.Server{
 		Handler:      s.Handler,
 		Addr:         httpAddr,
-		WriteTimeout: 15 * time.Second,
+		WriteTimeout: 5 * time.Minute,
```

Comment:
5 minutes? not agains it but why so much?

Reply:
https://github.com/goccy/bigquery-emulator/commit/65eafe1674ce5a97bfe5a6b87a503308b267f8a7 
apparently in cases when timeout happens job is created anyway and it's hard to recover from that state